airo-dataset-tools/docs/pose.md

```diff
@@ -0,0 +1,72 @@
+# Pose format
+
+Our main use case for storing poses, is for camera extrinsics or other 6D object poses.
+We value the following properties in a format:
+
+- **human friendly**: readable, easily interpretable and editable
+- can be understood as **little context** as possible
+- **hard to makes mistakes** when filling in the values
+- use of **SI units** (meters, radians, etc.)
+
+For this reason we decided to explicitly name all scalar fields (this way you always specify the correct amount).
+We also chose Euler angles for the rotation, because it is minimal (3 parameters), and gimbal lock is not an issue for storing poses.
+The json output of our Pydantic parser looks like this:
+
+```json
+{
+  "position_in_meters": {
+    "x": 0.0,
+    "y": 0.0,
+    "z": 0.15
+  },
+  "rotation_euler_xyz_in_radians": {
```

Comment:
are these the extrinsic X then Y then Z angles? or what is the euler-angle convention?